sdk/scheduler/src/test/java/com/mesosphere/sdk/state/StateStoreUtilsTest.java

```diff
@@ -154,7 +157,62 @@ public void testStateStoreWithMismatchedIdRaisesErrorOnStatus() {
         TaskStatus taskStatus = newTaskStatus(CommonIdUtils.toTaskId("not-" + taskName), TaskState.TASK_UNKNOWN);
 
         assertThat(taskInfo.getTaskId(), is(not(taskStatus.getTaskId())));
-        StateStoreUtils.getTaskInfo(stateStore, taskStatus);
+        StateStoreUtils.getTaskName(stateStore, taskStatus);
+    }
+
```

Comment:
@nickbp @gabrielhartmann @benclarkwood can we share our code style with each other and set format on save?